### PR TITLE
Updated Langchain to latest Version

### PR DESCRIPTION
**Pull request by Vikram-Ch14**

### CODE REVIEW

File: api/Pipfile
Comment (CodeCrafter7) [inline api/Pipfile:1]: @Vikram-Ch14 please remove the sentence-transformers = "==2.2.2" dependency from this file.
also remove the "==" from langchain version.

File: api/Pipfile.lock
Comment (CodeCrafter7) [inline api/Pipfile.lock:1]: remove this file from commit

**Review by CodeCrafter7 — CHANGES_REQUESTED** [review]

File: api/Pipfile.lock
Comment (Vikram-Ch14) [reply api/Pipfile.lock:1]: Removed Pipfile.lock file

File: api/Pipfile
Comment (Vikram-Ch14) [reply api/Pipfile:1]: done with above changes @CodeCrafter7

**Review by CodeCrafter7 — APPROVED** [review]

**Pull request merged**